rtrlib/pfx/pfx.h

```diff
@@ -175,5 +182,30 @@ void pfx_table_for_each_ipv4_record(struct pfx_table *pfx_table, pfx_for_each_fp
  */
 void pfx_table_for_each_ipv6_record(struct pfx_table *pfx_table, pfx_for_each_fp fp, void *data);
 
+/**
+ * @brief Copy content of src into dst
```

Comment:
nit picky, its not src and dst, should be `Copy content of @p src_table into @p dst_table`